invenio_oauthclient/contrib/github.py

```diff
@@ -112,11 +113,10 @@ def __init__(
             access_token_url="https://github.com/login/oauth/access_token",
             authorize_url="https://github.com/login/oauth/authorize",
             precedence_mask=precedence_mask,
+            signup_options=signup_options,
         )
 
-    def get_handlers(self):
-        """Return GitHub auth handlers."""
-        return dict(
+        self.handlers = dict(
```

Comment:
maybe we could add the `info_serializer` here as a config variable for the info handler?